xray/commands/audit/sca/yarn/yarn.go

```diff
@@ -83,6 +84,17 @@ func configureYarnResolutionServerAndRunInstall(params utils.AuditParams, curWd,
 		return runYarnInstallAccordingToVersion(curWd, yarnExecPath, params.InstallCommandArgs())
 	}
 
+	executableYarnVersion, err := biUtils.GetVersion(yarnExecPath, curWd)
```

Comment:
By convention, packages are given lower case, single-word names
So please change the `biUtils` to `biutils` in the import section.
```suggestion
	executableYarnVersion, err := biutils.GetVersion(yarnExecPath, curWd)
```